src/index.js

```diff
@@ -295,9 +297,10 @@ let Rosnodejs = {
         goal: { edges: 3,  radius: 1 } }));
    */
   getActionClient(options) {
-    options.nh = this.nh;
-    return new ActionClient(options);
+    return this.nh.actionClientInterface(options);
```

Comment:
This doesn't actually work. The [invoked function](https://github.com/RethinkRobotics-opensource/rosnodejs/blob/e68811c43a345782da863b7bd7c66498ad6d9729/src/lib/NodeHandle.js#L200) doesn't take an object as argument, it requires the server and type to be broken out. I'll create a PR to fix that.